### DISCUSSION TITLE
Contribution of KO to pathway

### DISCUSSION THREAD

**Discussion by rpey14** [cafferychen777/ggpicrust2#59] [Q&A] [answered]
Hi, 

Is there a way to create an output from ko2kegg_abundance that lists the contribution of each KO to assigned pathways? 

Thanks!

**Comment by cafferychen777** [comment, marked as answer]
Hi @rpey14 ,

Thank you for reaching out. I appreciate your interest in ggpicrust2 and your suggestion to include a feature that lists the contribution of each KO (KEGG Orthology) to assigned pathways in the `ko2kegg_abundance` function.

I haven't implemented this feature yet, but I do have plans to add it in a future version. While I can't provide an exact timeline for when this feature will be available, I will definitely consider it for the next release.

If you have any more questions or suggestions, please feel free to let me know. Your feedback is valuable in improving ggpicrust2.

Best regards,
